cfgov/unprocessed/css/on-demand/event.less

```diff
@@ -175,46 +174,92 @@
   vertical-align: middle;
 }
 
+.line-container {
+  margin: 0 0 unit(30px / @base-font-size-px, em);
+  border-top: 1px solid @gray-20;
+  border-bottom: 1px solid @gray-20;
 
-/* Event-Footer
-   ========================================================================== */
-/*
-.event-footer {
-    .respond-to-max(@bp-xs-max, {
-        a {
-            display: block;
-        }
+  &_body {
+    margin: unit(26px / @base-font-size-px, em) 0
+      unit(26px / @base-font-size-px, em);
+  }
 
-        .a-btn {
-            text-align: center;
-        }
+  &__no-top {
+    border-top: 0;
+    .line-container_body {
+      margin-top: 0;
+    }
+  }
+}
 
-        &_link {
-            padding: .625em 0;
+.modification-date {
+  .h6();
+  background: @green-20;
+  color: @black;
+  letter-spacing: 0;
+  padding: unit(1px / @base-font-size-px, em)
+    unit(10px / @base-font-size-px, em);
+  text-align: center;
+}
 
-            border-width: 1px 0;
-        }
-    });
+.post_heading {
+  .h2();
+  margin-bottom: unit(13px / @font-size, em);
+  .respond-to-min(@bp-sm-min, {
+    .h1();
+  });
+}
 
-    &_banner {
-        padding: 15px 20px;
+.post_dek {
+  .h4();
+  margin-bottom: unit(20px / @font-size, em);
+}
 
-        background-color: @gray-5;
+.post_body {
```

Comment:
Carry over TODO:

```
// TODO: `post_body` is used on article.html and events/_event-detail.html.
// Refactor to use DS classes if possible.
```

Reply:
Actually you removed `article.html`, so it can be:

```
// TODO: `post_body` is used on events/_event-detail.html.
// Refactor to use DS classes if possible.
```